### DISCUSSION TITLE
Compatibility 3R ECH2O - EHSXB08P50EF

### DISCUSSION THREAD

**Discussion by stefan-hausmeister** [raomin/ESPAltherma#484] [Q&A]
Hi there. Great project! 
I have a 3R ECH2O - EHSXB08P50EF heatpump and wanted to know if anyone has already experiences with this model?
If not, what would be best practice to make it work (didn't try so far)?

- select default header-configuration?
- or select of the already existing ECH2O?
- try & error by uncommenting definitions (anything I have to consider?)
- repeat?

Is this the way to go or is this project not working with the EHSX if there is no definition for it?
Of course I am happy to contribute a new definition file once it works.

Thanks in advance.

**Comment by jesseklm** [comment]
Hi, i would try this one: [Altherma(ERGA E EHSH-X P30-50 E_EF series 04-08kW-ECH2O).h](https://github.com/raomin/ESPAltherma/blob/main/include/def/Altherma(ERGA%20E%20EHSH-X%20P30-50%20E_EF%20series%2004-08kW-ECH2O).h)
